packages/app/src/api/core-domain/useCases/CreateOwnershipRequest.ts

```diff
@@ -0,0 +1,68 @@
+import { OwnershipRequest } from "@common/core-domain/domain/OwnershipRequest";
+import { OwnershipRequestStatus } from "@common/core-domain/domain/valueObjects/ownership_request/OwnershipRequestStatus";
+import { Siren } from "@common/core-domain/domain/valueObjects/Siren";
+import type { UseCase } from "@common/shared-domain";
+import { AppError } from "@common/shared-domain";
+import { ValidationError } from "@common/shared-domain/domain";
+import { Email } from "@common/shared-domain/domain/valueObjects";
+
+import type { IOwnershipRequestRepo } from "../repo/IOwnershipRequestRepo";
+
+interface Input {
+  askerEmail: string;
+  emails: string[];
+  sirens: string[];
+}
+
+export class CreateOwnershipRequest implements UseCase<Input, void> {
+  constructor(private readonly ownershipRequestRepo: IOwnershipRequestRepo) {}
+
+  public async execute({ sirens, emails, askerEmail }: Input): Promise<void> {
+    try {
+      if (!Array.isArray(sirens) || !Array.isArray(emails)) {
+        throw new ValidationError("Error for sirens or emails. Array is expected.");
+      }
+
+      const validatedAskerEmail = new Email(askerEmail);
+
+      const setSirens = new Set<Siren>();
+      const setEmails = new Set<Email>();
+
+      for (const siren of sirens) {
+        try {
+          const validatedSiren = new Siren(siren);
+          setSirens.add(validatedSiren);
+        } catch (error) {
+          console.error(`Error for Siren ${siren}`, error);
+        }
+      }
+
+      for (const email of emails) {
+        try {
+          const validatedEmail = new Email(email);
+          setEmails.add(validatedEmail);
+        } catch (error) {
+          console.error(`Error for email ${email}`);
```

Comment:
if error, should be save with error status, and this string error as error detail

Reply:
Will be closed in #1461